guidelines/sc/22/focus-appearance.html

```diff
@@ -13,14 +12,14 @@ <h4>Focus Appearance</h4>
 
     <p>Exceptions:</p>
     <ul>
-      <li>The focus indicator is determined by the <a>user agent</a> and cannot be adjusted by the author, or</li>
+        <li>The focus indicator is determined by the <a>user agent</a> and cannot be adjusted by the author, or</li>
         <li>The focus indicator and the indicator's background color are not modified by the author.</li>
     </ul>
-    
-    <p class="note">What is perceived as the user interface component or sub-component (to determine enclosure or size) depends on its visual <a>presentation</a>. The visual presentation includes the component's visible <a>content</a>, border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.</p>
+
+    <p class="note">What is perceived as the user interface component or sub-component (to determine the perimeter) depends on its visual <a>presentation</a>. The visual presentation includes the component's visible <a>content</a>, border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.</p>
```

Comment:
There are anchors (`<a>` here) that have no `href` so are not valid? Are these meant to be links to something?

Reply:
these get automagically converted to links to the definition when this gets published